packages/mobile/src/identity/verification.ts

```diff
@@ -63,23 +63,31 @@ export function* startVerification() {
 
   Logger.debug(TAG, 'Starting verification')
 
+  const startTime = Date.now()
+
   const { result, cancel, timeout } = yield race({
     result: call(doVerificationFlow),
     cancel: take(Actions.CANCEL_VERIFICATION),
     timeout: delay(VERIFICATION_TIMEOUT),
   })
 
   if (result === true) {
-    CeloAnalytics.track(CustomEventNames.verification_success)
+    ValoraAnalytics.track(AnalyticsEvents.verification_success, {
+      duration: Date.now() - startTime,
```

Comment:
Generally opposed to client side timing metrics but agree with the trade off to have these easily accessible

Reply:
Yeah, I was thinking of scrapping this because theoretically we can just calculate the different between the `verification_start` event and everything else. Decided to hold off on doing it until we have more bandwidth to focus on creating more complex data queries but think these timing metrics will be eliminated sometime soon